### ISSUE TITLE
What's the difference between A/MFCC→A and A/MFCC→AV in the paper?

### ISSUE THREAD

**Issue by PussyCat0700** [facebookresearch/av_hubert#81]
Hi Noir:

I am looking at Table 4 of "Learning audio-visual speech representation" and I am not sure what is the difference between A/MFCC->A(Hsu et al.) and A/MFCC->AV. To be more specific, could you please tell me which one requires audio-only pretraining in the penultimate iteration just as you mentioned in this issue #20  and which one refers to the AV-HuBERT model pretrained with just the original way?

Thanks a lot again, and I appreciate any help you may offer!

![Table content image](https://user-images.githubusercontent.com/75151459/210582154-ece693ec-2421-4945-9527-4d6521805a16.jpeg)

**Comment by PussyCat0700** [comment]
Checked #64 and #66 and this may be the difference:
1. Hsu et al. didn't apply modality dropout in pretraining, equivalent to p=0, a=1 version of AV-HuBERT in configuration.
2. Hsu et al. used audio-HuBERT's iter 4 cluster for last iteration's prediction, while A/MFCC->AV used iter 4 cluster generated from AV-HuBERT to perform the last iteration of pretraining.

Btw, Sec 4.5 of paper mentioned "pre-training an audio-HuBERT with
audio-visual cluster leads to better ASR performance (3.8%) than a pre-trained AV-HuBERT (4.6%)". Here, 3.8% is acquired by A/MFCC->AV in last row of Table 4, outperforming 4.6% of default-pretrained AV-HuBERT. A/MFCC->AV's outperforming AV-HuBERT is probably caused by different hyperparamter settings.
